src/containers/create-post/create-post.component.jsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from "react";
+import { addPost } from "database/addPost";
+
+const CreatePost = () => {
+  const [body, setBody] = useState("");
```

Comment:
Can use a better variable name than `body` and `opt1`, etc. Variables names aren't computationally expensive so they can be descriptive such as `postText` or `option1` for better readability

Reply:
done, works.